setupDB.js

```diff
@@ -0,0 +1,30 @@
+// JavaScript source code
+conn = new Mongo();
+db = conn.getDB("AnthroLinkDB");
+
+db.createUser(
+    {
+        user: "AnthroAdmin",
+        pwd: "password",
+
+        roles: [{ role: "userAdmin", db: "AnthroLinkDB" }]
+    }
+);
+
+db.createCollection("users");
+
+db = conn.getDB("admin");
+
+db.createUser(
+    {
+        user: "adminDB",
+        pwd: "password",
+
+        roles: [{ role: "userAdminAnyDatabase", db: "admin" }]
+    }
+);
+
+
```

Comment:
Remove Extra Lines below.

Reply:
I should leave a couple lines though, shouldn't I? The coding practices you've told me about before?